CONTRIBUTING.md

```diff
@@ -21,60 +22,63 @@ branch are tagged into a release monthly.
 
 1. Clone the repo into a public GitHub repository or [fork the repo](https://github.com/triggerdotdev/trigger.dev/fork). If you plan to distribute the code, keep the source code public to comply with the [Apache Licence 2.0](https://github.com/triggerdotdev/trigger.dev/blob/main/LICENSE).
 
-      ```
-      git clone https://github.com/triggerdotdev/trigger.dev.git
-      ```
-      > If you are on windows, run the following command on gitbash with admin privileges:
-      > `git clone -c core.symlinks=true https://triggerdotdev/trigger.dev.git`
+   ```
+   git clone https://github.com/triggerdotdev/trigger.dev.git
+   ```
+
+   > If you are on windows, run the following command on gitbash with admin privileges:
+   > `git clone -c core.symlinks=true https://triggerdotdev/trigger.dev.git`
+
 2. Navigate to the project folder
-      ```
-      cd trigger.dev
-      ```
+   ```
+   cd trigger.dev
+   ```
 3. Install the required packages using pnpm.
-      ```
-      pnpm i
-      ```
+   ```
+   pnpm i
+   ```
 4. Create your `.env` files
-      ```
-      cp .env.example .env && cp packages/database/.env.example packages/database/.env
-      ```
+   ```
+   cp .env.example .env && cp packages/database/.env.example packages/database/.env
+   ```
 5. Open the root `.env` file and fill in the required values Magic Link:
-            
-      Both of these secrets should be random strings, which you can easily generate (and copy into your pasteboard) with the following command:
-      
-      ```sh
-      openssl rand -hex 16 | pbcopy
-      ```
-
-      <p>Then set them here:</p>
-
-      ```
-      SESSION_SECRET=<string>
-      MAGIC_LINK_SECRET=<string>
-      ```
-      
-7. Start Docker. This starts the required services like Postgres.
-      ```
-      pnpm run docker
-      ```
-8. Migrate the database
-      ```
-      pnpm run db:migrate
-      ```
-9. Run the seed script
-      ```
-      pnpm run db:seed
-      ```
-5. Run the app. See the section below.
+
+   Both of these secrets should be random strings, which you can easily generate (and copy into your pasteboard) with the following command:
+
+   ```sh
+   openssl rand -hex 16 | pbcopy
+   ```
+
+     <p>Then set them here:</p>
+
+   ```
+   SESSION_SECRET=<string>
+   MAGIC_LINK_SECRET=<string>
+   ```
+
+6. Start Docker. This starts the required services like Postgres. If this is your first time using Docker, consider going through this [guide](docker-setup.md)
+   ```
+   pnpm run docker
+   ```
+7. Migrate the database
+   ```
+   pnpm run db:migrate
+   ```
+8. Run the seed script
+   ```
+   pnpm run db:seed
+   ```
+9. Run the app. See the section below.
 
 ## Running
 
 1. You can run the app with:
-      ```
-      pnpm run dev --filter webapp
-      ```
-      
-      It should run on port `3030`: [http://localhost:3030](http://localhost:3030/)
```

Comment:
These changes still need to be removed from this PR

Reply:
Oh! The whole contributing.md file?